person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @parent_permission = parent_permission
+    @name = name
+    @age = age
+    @id = 0
```

Comment:
[OPTIONAL] The ID of a person should be unique to the person😉. hence, giving it a default and the same value for all users defeats the purpose😉. you can do this instaad😉.

```suggestion
    @id = Random.rand(1..1000)
```